chore: add missing content about renamed plugins; fix links to the new books and remove hardcoded link to 1.2 docs (RHIDP-4293) (#570)
redhat-developer/red-hat-developers-documentation-rhdh: artifacts/rhdh-plugins-reference/argocd/argocd-plugin-admin.adoc

    plugins:
      - package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
        disabled: false
      - package: ./dynamic-plugins/dist/backstage-community-plugin-redhat-argocd
        disabled: false
----

[role="_additional-resources"]
.Additional resources

> * The package path, scope, and name of the {company} ArgoCD plugin has changed since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 

Reviewer: ```suggestion
* The package path, scope, and name of the {company-name} ArgoCD plugin has changed since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 
```

> * Note that the package path, scope, and name of the Red Hat ArgoCD plugin has changed since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 

Reviewer 2: ```suggestion
* The package path, scope, and name of the {company} ArgoCD plugin has changed since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 
```

Reviewer 3: Our attributes contain:
:company-name: Red Hat

Reviewer 3: ```suggestion
* The package path, scope, and name of the {company-name} ArgoCD plugin have been updated since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 
```

> * For more information on dynamic plugins, see link:{installing-and-viewing-dynamic-plugins-url}[{installing-and-viewing-dynamic-plugins-title}].

Reviewer 2: ```suggestion
* For more information on installing dynamic plugins, see link:{installing-and-viewing-dynamic-plugins-url}[{installing-and-viewing-dynamic-plugins-title}].
```

> * For more information on dynamic plugins, see link:{LinkPluginsGuide}#rhdh-installing-dynamic-plugins[Dynamic plugin installation] in the _{NameOfPluginsGuide}_ guide.

Reviewer 2: ```suggestion
* For more information on dynamic plugins, see the link:{LinkInstallingAndViewingDynamicPlugins}[{NameOfInstallingAndViewingDynamicPluginsGuide}] guide.
```

> * The package path, scope, and name of the {company-name} ArgoCD plugin has changed since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 
> 
> * For more information on dynamic plugins, see link:{LinkPluginsGuide}#rhdh-installing-dynamic-plugins[Dynamic plugin installation] in the _{NameOfPluginsGuide}_ guide.

Reviewer: ```suggestion
* The package path, scope, and name of the {company-name} ArgoCD plugin has changed since 1.2. For more information, see link:{release-notes-url}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 

* For more information on dynamic plugins, see link:{installing-and-viewing-dynamic-plugins-url}[{installing-and-viewing-dynamic-plugins-title}].
```

> * For more information on dynamic plugins, see link:{LinkPluginsGuide}#rhdh-installing-dynamic-plugins[Dynamic plugin installation] in the _{NameOfPluginsGuide}_ guide.

Reviewer 3: The added link returns 404. I'd suggest adding the following link:
```suggestion
* For more information on dynamic plugins, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/installing_and_viewing_dynamic_plugins/index[Installing and viewing dynamic plugins]
```

@Gerry-Forde We need to also include the attribute URLs for the new plugin titles to avoid using hardcoded links.

Reviewer 2: @hmanwani-rh I've also added a comment/suggestion for this ^^^
